init.ps1

```
@@ -1,54 +1,61 @@
+        return $true
+    } catch {
+        return $false
+    }
 }
 
-# Check if Scoop is installed
-if (Is-ScoopInstalled) {
-    Write-Host "Scoop is already installed."
-}
-else {
-    Write-Host "Scoop is not installed. Installing Scoop..."
-    Invoke-RestMethod -Uri https://get.scoop.sh | Invoke-Expression
+# Check if Node.js is installed
+if (Is-NodeInstalled) {
+    Write-Host "Node.js is already installed."
+} else {
+    Write-Host "Node.js is not installed. Installing Node.js via Scoop..."
+    if (-not (Get-Command scoop -ErrorAction SilentlyContinue)) {
+        Write-Host "Scoop not found. Installing Scoop first..."
+        Invoke-RestMethod -Uri https://get.scoop.sh | Invoke-Expression
+    }
+    scoop install nodejs
 }
 
-# Verify installation
-if (Is-ScoopInstalled) {
-    Write-Host "Scoop has been successfully installed."
-}
-else {
-    Write-Host "Failed to install Scoop."
-}
 
+# Pin Appwrite server version
+$serverVersion = "1.7.4"
+Write-Host "Detected Appwrite server version: $serverVersion"
+
+# Install Appwrite CLI via npm (version pinned)
+Write-Host "Installing Appwrite CLI via npm (version pinned)"
+npm install -g appwrite@7.0.0
 
-Write-Host "Install Appwrite-cli using Scoop"
-scoop install https://raw.githubusercontent.com/appwrite/sdk-for-cli/master/scoop/appwrite.config.json
 
 docker run -it --add-host host.docker.internal:host-gateway --rm `
     --volume /var/run/docker.sock:/var/run/docker.sock `
     --volume "$(pwd)/appwrite:/usr/src/code/appwrite:rw" `
     --entrypoint="install" `
-    appwrite/appwrite:1.7.4
+<<<<<<< HEAD
+    appwrite/appwrite:latest
+=======
+    appwrite/appwrite:$serverVersion
+>>>>>>> f143dbf (Fix init.sh and init.ps1 for Windows and WSL compatibility)
 
 $projectId = "resonate"
 
-# Remove previous Appwrite Cli data
-Remove-Item -Recurse -Force $HOME\.appwrite
+# Remove previous Appwrite CLI data
+Remove-Item -Recurse -Force $HOME\.appwrite -ErrorAction SilentlyContinue
```

Don't silently continue on Errors